src/main/java/us/ihmc/zed/SL_IMUData.java

```diff
@@ -49,7 +49,7 @@ public class SL_IMUData extends Pointer {
 	*/
 	public native @ByRef SL_Vector3 angular_velocity(); public native SL_IMUData angular_velocity(SL_Vector3 setter);
 	/**
-	\brief Linear acceleration vector (3x1) of the gyroscope in m/s².
```

Comment:
Huh, it lost the special characters

Reply:
Yeah I don't really know why it happened and I guess it's not so important